universum/modules/vcs/github_vcs.py

```diff
@@ -17,7 +21,34 @@ def get_time():
     return datetime.datetime.utcnow().replace(microsecond=0).isoformat() + "Z"
 
 
-class GithubMainVcs(ReportObserver, git_vcs.GitMainVcs):
+class GithubToken(Module):
+
+    @staticmethod
+    def define_arguments(argument_parser):
+        parser = argument_parser.get_or_create_group("GitHub", "GitHub repository settings")
+
+        parser.add_argument("--github-integration", "-ght", dest="integration_id", metavar="GITHUB_INTEGRATION",
+                            help="GitHub application ID (see 'general')")
+        parser.add_argument("--github-installation", "-ghs", dest="installation_id", metavar="GITHUB_INSTALLATION",
+                            help="in-project installation ID (see 'integrations & services')"
+                                 " - not needed for 'github-handler'")
```

Comment:
If the handler doesn't require this parameter we need to remove it.
Note we already have number of modes of operation with excessive parameters.
We'd better not increase this number further.

Reply:
???
GitHub Handler is not the only user of this arguments
While Handler can get the installation ID out of payload, Universum should still receive it for report for review somehow

Reply:
Then we need to split GithubToken class